packages/react-form-state/src/FormState.ts

```diff
@@ -152,9 +151,9 @@ export default class FormState<
       })
       .filter(error => {
         return error.field != null && error.message != null;
-      }) as ClientError[];
+      });
 
-    return [...errors, ...fieldErrors];
+    return fieldErrors.length + errors.length;
```

Comment:
Is it possible to have an error counted twice here?

Reply:
doesn't actually matter, all we care about is that it is greater than 0

Reply:
Can maybe just roll this logic into `valid` so it's more obvious that that's what matters